<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeechTranscriptionWidget</class>
 <widget class="QWidget" name="SpeechTranscriptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetUpperPanel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <widget class="QLineEdit" name="lineEditCursorFirstDocPosX">
      <property name="geometry">
       <rect>
        <x>398</x>
        <y>19</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>###</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCursorFirstFrameInd">
      <property name="geometry">
       <rect>
        <x>398</x>
        <y>-1</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>###</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>DocPosX</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Marker Level</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="radioButtonPhoneLevel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>26</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonWordLevel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>11</y>
         <width>49</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Word</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>2</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Marker ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMarkerId">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>2</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>###</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxCurMarkerStopOnPlayback">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Stops Playback</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>363</x>
        <y>0</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cursor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCursorSecondFrameInd">
      <property name="geometry">
       <rect>
        <x>484</x>
        <y>-1</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>###</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCursorSecondDocPosX">
      <property name="geometry">
       <rect>
        <x>484</x>
        <y>19</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>###</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCursorFramesCount">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>-1</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>###</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCursorDocLen">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>19</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>###</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxLang">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="radioButtonLangRu">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>0</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ru</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonLangUk">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>uk</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonLangNone">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBoxSpeakerId">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditMarkerText">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>annotation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditMarkerPhoneList">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A N N O T A1 T1 I O N</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollBar" name="horizontalScrollBarSamples">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SpeechTranscriptionPaintWidget" name="widgetSamples" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpeechTranscriptionPaintWidget</class>
   <extends>QWidget</extends>
   <header>SpeechTranscriptionPaintWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
